plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2192,6 +2235,20 @@ private static Map<ColumnStatisticMetadata, Block> getColumnStatistics(Map<List<
                 .orElse(ImmutableMap.of());
     }
 
+    private static Map<ColumnStatisticMetadata, Block> getColumnStatistics(
```

Comment:
Can you try merging these two `getColumnStatistics` methods into one? I suppose having similar methods will confuse code readers in the future.

Reply:
I think this next state should look better